apps/publisher/src/app.tsx

```diff
@@ -92,7 +92,7 @@ function App() {
     // TODO: per-stream audio/video toggling
     // toggleAudio,
     // toggleVideo,
-  } = useMultiMediaDevices();
+  } = useMultiMediaStreams();
```

Comment:
We should not rename it to `useMultiMediaStreams` and let it manage other streams. This hook should not manage the streams, instead, it just provides some functions to manipulate the user's camera/microphones and screen capture.

Reply:
Let's merge it and refine it in #241